src/hooks/useGetDocumentTitlePrice.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+import useGetPriceData from './useGetPriceData'
+
+const useGetDocumentTitlePrice = () => {
+  const priceData = useGetPriceData()
+  const cakePriceUsdNumber = priceData ? Number(priceData.prices.Cake) : Number.NaN
```

Comment:
same feedback than on the FE. Don't use Number

Reply:
done, i added bignumber library